tweetcloud/util.py

```diff
@@ -0,0 +1,49 @@
+import contextlib
+import itertools
+import sys
+import threading
+import time
+from typing import Final, Generator, Iterator, Optional, TextIO
+
+
+@contextlib.contextmanager
+def spinner(msg: Optional[str] = None, new_line: bool = True, done=True) -> Generator:
```

Comment:
This is awesome that you wrote this, but should we just use a module like https://github.com/manrajgrover/py-spinners?

Reply:
oh yeah, or something like this https://github.com/rsalmei/alive-progress

I just had it sitting around and didn't think too much about it. I'll update